packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -300,6 +300,7 @@ export interface PluginPackageLanguageContribution {
     aliases?: string[];
     mimetypes?: string[];
     configuration?: string;
+    icon: IconUrl;
```

Comment:
Suggestion: I don't think this property is mandatory. Missing a `?`?

Reply:
You are absolutely right! Thank you for catching that!